invenio_records_files/views.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2019 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio-Records-Files REST integration."""
+
+from __future__ import absolute_import, print_function
+
+from flask import Blueprint, abort, g
+from invenio_files_rest.views import bucket_view, object_view
+from six import iteritems
+
+from invenio_records_files.models import RecordsBuckets
+
+
+def create_blueprint_from_app(app):
+    """Create Invenio-Records-Files blueprint from a Flask application.
+
+    :params app: A Flask application.
+    :returns: Configured blueprint.
+    """
+    records_files_blueprint = Blueprint(
+        'invenio_records_files',
+        __name__,
+        url_prefix='')
+
+    for config_name, rec_types in \
+            iteritems(app.config['RECORDS_FILES_REST_ENDPOINTS']):
+        files_resource_name = app.config['RECORDS_FILES_ENDPOINT_SUFFIX']
+        for rec_type in rec_types:
+            record_item_path = app.config[config_name][rec_type]['item_route']
+            records_files_blueprint.add_url_rule(
+                '{record_item_path}/{files_resource_name}'.format(**locals()),
```

Comment:
This (added by me) is going to break badly if people sets `RECORDS_FILES_ENDPOINT_SUFFIX` to something like `/myfiles`, we should properly join these URLs. Watch out with [`urllib.parse.url_join`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) which won't work either:
```console
In [1]: from urllib.parse import urljoin
In [2]: urljoin('/api/records/{}/'.format(1), '/files')
Out[2]: '/files'
```